include/libdnf/module/module_item.hpp

```diff
@@ -192,7 +192,8 @@ class ModuleItem {
     std::string get_name_stream() const;
     // @replaces libdnf:module/ModuleItem.hpp:method:ModuleItem.getNameStreamVersion()
     std::string get_name_stream_version() const;
-    std::string get_name_stream_context() const;
+    std::string get_name_stream_staticcontext() const;
```

Comment:
May I ask you for a description? Because static_context is a boolean option in metadata, it could be confusing without the description.

Reply:
Oh, right, I forgot it's named the same. But at least both these methods that operate with the computed static contexts are private.